templates/php-component/tests/phpunit/bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
```

Comment:
nemohlo by to byt jen v adresari /tests/  ?

Reply:
Pokud tam chceme i ty funkční testy, tak mi přijde, že to patří sem.